### PR TITLE
Pushing new bump test code with plot improvements

### PR DESCRIPTION
**Pull request by craiglagegit**
Plotting improvements include adding vertical lines where the pass/fail decision is made and adding a PASS/FAIL text to the plot.

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb [hunk 1]
```diff
@@ -4,8 +4,10 @@
    "cell_type": "markdown",
```
Comment (ykangastro) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #2.    data_dir = "/home/c/cslage/u/MTM1M3/data/"
</pre><p>This line might not be necessary. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/87/discussion/#comment-1393424432'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (ykangastro) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #4.    id = 108
</pre><p>Question: it is an id for an actuator which we wanna check, right? </p><p>If it is maybe you'd better add a new cell and put comment like </p><p>"put the number of actuator to see the result from bump test"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/87/discussion/#comment-1393424434'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (ykangastro) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #7.    plt.savefig(data_dir + f"Bump_Test_Following_Errors_{id}.png")
</pre><p>I think you could create new directory/use exist directory</p><p>1) os.makedirs(<em>directory_name_here</em>, exist_ok=True)</p><p>2) fig.savefig(f"directory_name_here/Bump_Test_Following_Errors_{id}.png")</p><p>: either making new director or just save the file there if there is already directory.</p><p>(e.g. directory_name_here can be "plots" or "result_from_SITCOM818", etc).</p><p>note that os is already imported. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/87/discussion/#comment-1393424435'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (ykangastro) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #2.    pdf = PdfPages(data_dir + f"/Bump_Test_Following_Errors_{timestamp}.pdf")
</pre><p>Same as previous review/comment. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/87/discussion/#comment-1393424436'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (craiglagegit) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: I fixed this with a local plots directory.

Comment (craiglagegit) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: I don't think this is necessary.  I put in a comment to make it clearer.

Comment (craiglagegit) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: Yes.  That's basically what I did.

Comment (craiglagegit) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: I fixed this one, too.

Comment (pkubanek) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-818_SITCOMTN-083.ipynb:4]: if data_dir is os.Path, you can leave leading /

**Review by ykangastro — APPROVED** [review]
Approved!

**Comment by craiglagegit** [comment]
Merged and branch deleted.

**Pull request merged**